src/LuaLib.ts

```diff
@@ -87,6 +87,7 @@ export enum LuaLibFeature {
     Unpack = "Unpack",
 }
 
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Why is it complaining here, because of the key names?

Reply:
Yes PascalCase key names are not allowed. I didn't change them to camelCase because I wasn't sure what the implications of that would be.